packages/policy/src/policy.ts

```diff
@@ -0,0 +1,270 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This class provides scannng of repositories to ensure they follow the
+ * guidance listed in go/cloud-dpe-oss-standards. It can be run locally as
+ * a CLI:
+ *
+ *  $ policy --repo googleapis/sloth
+ *
+ * Or it can be used as an export to a Google Sheet that's used to construct
+ * a dashboard (go/yoshi-live in this case)
+ */
+
+import {request} from 'gaxios';
+import {operations} from '@octokit/openapi-types';
+import {Octokit} from '@octokit/rest';
+
+export type GitHubRepo = operations['repos/get']['responses']['200']['content']['application/json'];
+export const githubRawBase = 'https://raw.githubusercontent.com';
+
+export interface PolicyResult {
+  repo: string;
+  language: string;
+  renovate: boolean;
```

Comment:
What do the booleans in this interface mean?

Reply:
Good question.  I am renaming these, and adding explanations to each one.